examples/nas/cifar10_tutorial.py

```diff
@@ -0,0 +1,51 @@
+"""
+Run NAS baseline methods
+========================
+We provide 4 NAS baseline methods now, the default one is bayesian optimization.
+Here is a tutorial about running NAS baseline methods.
+
+Generally, to run a non-default NAS methods, we will do the following steps in order:
+1. Prepare the dataset in the form of torch.utils.data.DataLoader.
+2. Initialize the CnnModule/MlpModule with the class name of the NAS Searcher.
+3. Start search by running fit function.
+Refer the cifar10 example below for more details.
+"""
+import numpy as np
+import torch
+import torchvision
+import torchvision.transforms as transforms
+from torch.nn.functional import cross_entropy
+
+from autokeras import CnnModule
+from autokeras.nn.metric import Accuracy
+from nas.greedy import GreedySearcher
+
+if __name__ == '__main__':
+    transform = transforms.Compose(
+        [transforms.ToTensor(),
+         transforms.Normalize((0.5, 0.5, 0.5), (0.5, 0.5, 0.5))])
```

Comment:
Please see some examples online. I think they normalized with a different value.
They also did some data augmentation.
Refer to this example.
https://github.com/kuangliu/pytorch-cifar/blob/master/main.py